test/integration-test.py

```diff
@@ -241,6 +245,52 @@ def run_client_tests():
     cmd = os.path.join(root, 'tests', 'boulder-integration.sh')
     run_custom(cmd, cwd=root)
 
+
+def get_future_output(cmd, date, cwd=None):
+    return subprocess.check_output('FAKECLOCK=`date -d "%s"` %s' % (date.isoformat(), cmd), cwd=cwd, shell=True)
```

Comment:
This is a lot of formatting back and forth. Let's do:

`return subprocess.check_output(cmd, cwd=cwd, shell=True, env={'FAKECLOCK': date.strftime("%s")})`